src/tools/tidy/src/pal.rs

```diff
@@ -62,6 +62,8 @@ const EXCEPTION_PATHS: &[&str] = &[
     "library/std/src/panic.rs",   // fuchsia-specific panic backtrace handling
     "library/std/src/personality.rs",
     "library/std/src/personality/",
+    "library/std/src/thread/local.rs", // UEFI specific thread_local implementation
+    "library/std/src/thread/mod.rs",   // UEFI specific thread_local implementation
```

Comment:
I initially only corrected your spelling here but I have been made aware that there is active work on reducing this list of exceptions by @devsnek. Rapidly. For example:
- https://github.com/rust-lang/rust/pull/108485
- https://github.com/rust-lang/rust/pull/108796
- https://github.com/rust-lang/rust/pull/108798

In order to avoid merge conflicts and in general work collisions, please introduce what factorization you need to pull this code out into `std::sys`, instead. It may be better to open another PR for this since you should be able to do the first pass of this without extending the UEFI implementation. I am sure @devsnek can advise you if you need help. Feel free to r? me if you do open another PR, it should be comparatively quick to review/approve.

Reply:
Fixed by https://github.com/rust-lang/rust/pull/108927